lib/ebanq_api/client.rb

```diff
@@ -59,8 +60,52 @@ def profile
       @profile ||= EbanqApi::Profile.new(self)
     end
 
+    # Declares an accounts instance.
+    def accounts
+      @accounts ||= EbanqApi::Accounts.new(self)
+    end
+
+    # Declares an news instance.
+    def news
+      @news ||= EbanqApi::News.new(self)
+    end
+
+    # Declares an messages instance.
+    def messages
+      @messages ||= EbanqApi::Messages.new(self)
+    end
+
     private
 
+    def get(path, params, headers)
+      RestClient::Request.execute(method: :get, url: path,
+                                  timeout: 10,
+                                  headers: headers
+                                             .merge!(params: params))
+    end
+
+    def post(path, params, headers)
+      RestClient.post(path, params, headers)
+    end
+
+    def delete(path, headers)
+      RestClient.delete path, headers
+    end
+
+    def process_raw_response(response)
+      !response.body.empty? ? process_response_body(response) : response.body
```

Comment:
method should return data in one format (but response.body could be an empty string)